sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java

```diff
@@ -243,6 +246,18 @@ public void setMenuCaption(String caption) {
 	}
 
 	/**
+	 * Sets the title text for the menu
```

Comment:
This javadoc block is not really matching what the method itself is doing. also missing @param for the resource parameter.